src/network/networkd-routing-policy-rule.c

```diff
@@ -1016,11 +1025,13 @@ int manager_rtnl_process_rule(sd_netlink *rtnl, sd_netlink_message *message, Man
                 return 0;
         }
 
-        r = sd_netlink_message_read_u8(message, FRA_L3MDEV, &tmp->l3mdev);
+        uint8_t l3mdev;
```

Comment:
this should be zero initialized, no? because if sd_netlink_message_read_u8() fails with ENODATA the variable will not be initialized